src/core/i18n/translationprocessor.js

```diff
@@ -45,7 +45,7 @@ export default class TranslationProcessor {
   }
 
   static _interpolate (stringToInterpolate, interpolationParams) {
-    const interpolationRegex = new RegExp(/\[\[([a-zA-Z0-9]+)\]\]/, 'g');
```

Comment:
Is this our only use of the `RegExp` constructor in the SDK? If not, we may need to update those other instances as well.

Reply:
it's the only usage in the actual source code (there's a few usages in conf but ie11 doesn't care about that)